cmd/tool/matrix/matrix.go

```diff
@@ -218,6 +238,23 @@ func minBucket(buckets []bucket) int {
 	return n
 }
 
+func isAllBucketsUsed(buckets []bucket) bool {
+	for _, b := range buckets {
+		if b.Total == 0 {
+			return false
+		}
+	}
+	return true
+}
+
+func consistentBucket(pkg string, n uint) int {
+	h := md5.New()
+	io.WriteString(h, pkg)
+	hash := fmt.Sprintf("%x", h.Sum(nil))
+	decimal, _ := strconv.ParseUint(hash[:10], 16, 64)
+	return int(decimal) % int(n)
```

Comment:
Maybe use `maphash.Hash{}` ?
```suggestion
var hashSeed = maphash.MakeSeed()

func consistentBucket(pkg string, n uint) int {
	return int(maphash.String(hashSeed, str)) % int(n)
```

Maphash was designed specifically for this case.

Props: 
1. fastest hash from standart library.
2. zero allocation.

Thanks!